kuma/static/js/wiki-helpful-survey.js

```diff
@@ -1,17 +1,16 @@
-(function(ga, $, mdn) {
+(function($, mdn) {
     'use strict';
 
     var $survey;
     var $votes;
-    var gaInterval;
     var gaChecks = 0;
 
     // ga loads async, so there's a chance it wont be available when this
     // script is parsed. we'll give it 2 seconds.
     function checkGA() {
         // survey won't do much good if we can't store the data
         // checking for ".create" due to Ghostery mocking of ga
-        if (ga && ga.create) {
+        if (window.ga && ga.create) {
```

Comment:
I believe it does not go on to check the second condition if the first fails and accessing ga globally if the first passes is not a problem. I could not explain why this is.